performance-tests/remotewb-performance-tests/addUsers.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+$JBOSS_HOME/bin/add-user.sh -a perfUser 'perfUser1234;'
+$JBOSS_HOME/bin/add-user.sh -a engUser 'engUser1234;'
+echo "perfUser=admin" >> $JBOSS_HOME/standalone/configuration/application-roles.properties
+echo "engUser=admin,engineering" >> $JBOSS_HOME/standalone/configuration/application-roles.properties
```

Comment:
Ok, I will do something with it. However, deploy.sh will be more complicated when some processes may stuck and then it is not possible to redeploy the deployment unit.